<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTable</class>
 <widget class="QWidget" name="DataTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableData"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Feature">
       <property name="text">
        <string>Feature</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFeature"/>
     </item>
     <item>
      <widget class="QLabel" name="labelFilter">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFilter"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableDataDescription"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
